lib/components/dashboard/Actionbar.tsx

```diff
@@ -39,9 +39,9 @@ import { Typography } from '@mui/material';
 import Box from '@mui/material/Box';
 import * as amplitude from '@amplitude/analytics-browser';
 
-const majorOptions = allMajors.map((major) => ({
-  abbrev: major.abbrev,
-  name: major.degree_name,
+const majorOptions = majorNames.map((major) => ({
```

Comment:
majorNames is an array of major names (also _ids)
majorInfos has url, abbrev, and number of credits so that we don't have to query the db for a major unless it's selected in a plan